Extension/CHANGELOG.md

```diff
@@ -3,11 +3,12 @@
 ## Version 0.16.0: March 15, 2018
 * Enable autocomplete for local and global scopes. [#13](https://github.com/Microsoft/vscode-cpptools/issues/13)
 * Support additional multiline comment patterns. [#1100](https://github.com/Microsoft/vscode-cpptools/issues/1100), [#1539](https://github.com/Microsoft/vscode-cpptools/issues/1539)
-  * Add new setting: `C_Cpp.commentContinuationPatterns`.
+  * Adds new setting: `C_Cpp.commentContinuationPatterns`.
```

Comment:
"Adds" doesn't match the other usages of "Add"?

Reply:
I made the lines more consistent